lib/df-detection.c

```diff
@@ -159,7 +158,8 @@ int filter_stack(const unsigned long stack_entries[], int num_entries)
 		if (strnstr(buf, "copy_from_user", len) ||
 		    strnstr(buf, "copyin", len) ||
 		    strnstr(buf, "strncpy_from_user", len) ||
-		    strnstr(buf, "get_user", len))
+		    strnstr(buf, "get_user", len) ||
```

Comment:
By the way, do you actually see occurrences get_user() in the stacks? I've never seen them - probably because that's a macro, not a function.

Reply:
some Intersection BUGS are printed during rebooting in syzkaller contain get_user in its stack system calls were (ioctl, execve), but i am not pretty sure is there any crashes detected after rebooting have get_user in its stack i can know that after disabling DF_DETECTION during rebooting.